lib/ansible/cli/doc.py

```diff
@@ -150,7 +150,7 @@ def run(self):
 
         return 0
 
-    def get_all_plugins_of_type(self, plugin_type):
+    def get_all_plugins_of_type(self, plugin_type, loader=False):
```

Comment:
Since loader is unconditionallt defined via a local variable on the next line, 154, I don't believe this is the correct fix.  Probably, the caller of this function needs to be changed so it doesn't send loader instead.

Reply:
I checked if proposed changes broke any other feature, but it did not, plus it fixes the concerned issue too. but I agree with your suggestion, I'll resubmit changes considering your suggestion.  Thanks for review.